src/main/kotlin/org/neo4j/graphql/CrudOperations.kt

```diff
@@ -34,6 +35,35 @@ fun augmentedSchema(ctx: Translator.Context, type: ObjectTypeDefinition): Augmen
     } else result
 }
 
+fun augmentedSchema(ctx: Translator.Context, source: ObjectTypeDefinition, target: ObjectTypeDefinition): Augmentation? {
+    val sourceIdField = source.fieldDefinitions.find { it.type.name() == "ID" }
+    val targetIdField = target.fieldDefinitions.find { it.type.name() == "ID" }
+    if (sourceIdField == null || targetIdField == null) {
+        return null
+    }
+    val sourceTypeName = source.name
+    val targetTypeName = target.name
+    if (ctx.mutation.enabled && !ctx.mutation.exclude.contains(sourceTypeName)
+            && !ctx.mutation.exclude.contains(targetTypeName)) {
+        val targetField = source.fieldDefinitions
+                .filter {
+                    val type = when(it.type) {
+                        is ListType -> it.type.inner()
+                        else -> it.type
+                    }
+                    type.name() == targetTypeName
+                }
+                .firstOrNull() ?: return null
+        val targetFieldName = targetField.name.capitalize()
+        val targetIDStr = if (targetField.type is ListType) "[ID]!" else "ID!"
```

Comment:
Not sure if this is common or actually used, and not rather a list of Id's pointing elsewhere?

Reply:
I used the same convention as explained here:
https://github.com/neo4j-graphql/neo4j-graphql#auto-generated-mutations